packages/chakra-ui-core/src/CAvatar/tests/CAvatar.test.js

```diff
@@ -1,45 +1,85 @@
 import { CAvatar, CAvatarBadge } from '..'
-import { render, fireEvent } from '@/tests/test-utils'
+import { render, wait, waitMs, screen } from '@/tests/test-utils'
+const LOAD_FAILURE_SRC = 'LOAD_FAILURE_SRC'
+const LOAD_SUCCESS_SRC = 'LOAD_SUCCESS_SRC'
+
+beforeAll(() => {
+  process.browser = true // Mock process.browser for CAvatar created()
+})
+
+beforeAll(() => {
+  // Mock Img
+  // eslint-disable-next-line accessor-pairs
+  Object.defineProperty(global.Image.prototype, 'src', {
+    set (src) {
+      // Mock the Image.prototype.onload/onerror implementations
+      this.onload = jest.fn().mockImplementation(fn => fn)
+      this.onerror = jest.fn().mockImplementation(err => err)
```

Comment:
Had to mock the `Image.onload` implementation because it was failing on my machine. It said that `this._onload` is not a function. Here's the error message:
![image](https://user-images.githubusercontent.com/21237954/87244956-e1fda100-c473-11ea-95bb-490d211dfc80.png)


Mocking it fixed it. 👍🏽

Reply:
I think this is a false positive. If you mock onload and onerror tests would fail. Because of `hasLoaded` in avatar and image components. 

Yep, as you can see you didn't mock the onload and onerror:

![image](https://user-images.githubusercontent.com/342666/87245311-16aa3200-c44d-11ea-9051-f080619c482a.png)

Reply:
i think your jest configuration is failing. Do you have jest or jsdom installed globally if so please remove it.

And can you remove and fresh install node_modules?
Seems like this line doesnt work on your machine and issue might be related to jsdom or jest version.
https://github.com/chakra-ui/chakra-ui-vue/blob/afaa8bc8f8eb06f057fc91bb40541047229da4da/jest.config.js#L33

Reply:
Let me try a fresh raw instal.

Reply:
btw we shoud not mock `this.onload` and `this.onerror` thats why im trying to find the issue on your machine.

Reply:
turns out this is not related to jest or jsdom so its good 😄